core.cfg

```diff
@@ -6,7 +6,7 @@ extends =
     versions.cfg
     tests.cfg
 
-parts +=
+parts =
```

Comment:
@pbauer the inclusion of tests.cfg above is now almost obsolete - the parts defined in tests.cfg (test, robot) are overwritten by this section.
Is that intentional or should we re-add test + robot here?